### PR TITLE
How to use IPv6 in Docker using Proxmox and pfSense

### PR DESCRIPTION
**Pull request by bruj0**

### CODE REVIEW

**Comment by bruj0** [comment]
Slightly updated version: https://github.com/bruj0/docker_ipv6_securely
I'll update it with the template once I have some feedback, thanks!

**Comment by n-se** [comment]
What exactly should we review now? Your submitted tutorial or your link to the other tutorial?

If we should review https://github.com/bruj0/docker_ipv6_securely please update your pull request or submit a new pull request.

**Comment by bruj0** [comment]
I've updated and modified this commit to be better close to the template tutorial.
I'm also adding the disclaimer:

I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Rodrigo A. Diaz Leven <ramakandra@gmail.com>

To answer your question this PR is now ready for review.

File: tutorials/docker-ipv6-securely/01.en.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+* z: x:y:w::30:1/76 VIP gateway for VM2
+  * z: x:y:w:31::1/80 for Docker containers in VM2
+* z: x:y:w:3e80::1/76 VMx
+  * z: x:y:w:3e81::1/80 for Docker containers in VMx
+
+## Hypervisor
+We will need a way to provision VMs, for this article we selected Proxmox
+More information: 
+* https://www.proxmox.com/en/proxmox-ve/get-started
+* How to install it remotely on any server: https://www.linkedin.com/pulse/installing-any-os-headless-server-rodrigo-leven/
```
Comment (n-se) [inline tutorials/docker-ipv6-securely/01.en.md]:
```suggestion
* How to install it remotely on any server: https://community.hetzner.com/tutorials/install-and-configure-proxmox_ve
```

File: tutorials/docker-ipv6-securely/01.en.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+![Proxmox](images/proxmox.png)
+
+
+### Bridges
+We will create 3 bridges:
+* **vmbr0**: It will connect to the internet and receive both IPv4 and IPv6 traffic but will only have the main IPv4 assigned xxx.yyy.145.162/26
+* **vmbr1**: The internal network shared between VMs, 10.x.x.x
+* **vmbr2**: This bridge will hold the IPv6 network for our VMs, in this case a /64
+
```
Comment (n-se) [inline tutorials/docker-ipv6-securely/01.en.md]: Could you also show us ``/etc/network/interfaces`` for this configuration?

File: tutorials/docker-ipv6-securely/01.en.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+SPDX-License-Identifier: MIT
+path: "/tutorials/docker-ip6-securely"
+slug: "docker-ip6-securely"
+date: "2019-06-16"
+title: "IPv6 on docker securely"
+short_description: "This tutorial will help you setup docker with IPv6 in a secure way."
+tags: ["docker", "ipv6", "proxmox"]
+author: "Rodrigo A. Diaz Leven"
+author_link: "https://github.com/bruj0/"
+author_img: "https://avatars1.githubusercontent.com/u/2046474?s=460&v=4"
```
Comment (n-se) [inline tutorials/docker-ipv6-securely/01.en.md]:
```suggestion
author_img: "https://avatars1.githubusercontent.com/u/2046474"
```

File: tutorials/docker-ipv6-securely/01.en.md (hunk 2 above)
Comment (bruj0) [reply tutorials/docker-ipv6-securely/01.en.md]: Added the contents to the article.

**Pull request merged**